docs/web3-eth.rst

```diff
@@ -1512,3 +1512,82 @@ Example
     > "Mist/v0.9.3/darwin/go1.4.1"
 
 ------------------------------------------------------------------------------
+
+.. _eth-getProof:
+
+getProof
+========
+
+.. code-block:: javascript
+
+    web3.eth.getProof(address, storageKey, blockNumber, [callback])
+
+Returns the account and storage-values of the specified account including the Merkle-proof as described in `EIP-1186 <https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1186.md>`_.
+
+----------
+Parameters
+----------
+
+1. ``String`` 20 Bytes:  The Address of the account or contract.
+2. ``Array`` 32 Bytes: Array of storage-keys which should be proofed and included. See :ref:`web3.eth.getStorageAt <eth-getStorageAt>`.
+3. ``Number | String | "latest" | "earliest"``: Integer block number, or the string "latest" or "earliest".
+4. ``Function`` - (optional) Optional callback, returns an error object as first parameter and the result as second.
+
+-------
+Returns
+-------
+
+``Promise<Object>`` - A account object.
+
+    ``balance`` - The balance of the account. See :ref:`web3.eth.getBalance <eth-getBalance>`.
+    ``codeHash`` -  hash of the code of the account. For a simple Account without code it will return "0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
+    ``nonce`` -  Nonce of the account.
+    ``storageHash`` - SHA3 of the StorageRoot. All storage will deliver a MerkleProof starting with this rootHash.
+    ``accountProof`` - Array of rlp-serialized MerkleTree-Nodes, starting with the stateRoot-Node, following the path of the SHA3 (address) as key.
+    ``storageProof`` - Array of storage-entries as requested.
+    ``key`` -  The requested storage key.
+    ``value`` -  The storage value.
+
+-------
+Example
+-------
+
+
+.. code-block:: javascript
+
+    web3.eth.getProof(
+        "0x1234567890123456789012345678901234567890",
+        ["0x0000000000000000000000000000000000000000000000000000000000000000","0x0000000000000000000000000000000000000000000000000000000000000001"],
+        "latest"
+    ).then(console.log);
+    > {
+        "jsonrpc": "2.0",
+        "id": 1,
+        "result": {
```

Comment:
It should just return the mapped result object from the JSON-RPC response

Reply:
Yes.